ttnn/cpp/ttnn/operations/conv/conv2d/conv2d_pybind.hpp

```diff
@@ -0,0 +1,18 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "ttnn/cpp/pybind11/decorators.hpp"
+#include "ttnn/operations/conv/conv2d/conv2d.hpp"
+#include "ttnn/types.hpp"
+
+#include "conv2d.hpp"
```

Comment:
Please review ccl/all_gather example again. It shows how to avoid these heavy includes

Reply:
Changed.